### PR TITLE
Feature/add estimated in table

### PR DESCRIPTION
**Pull request by olsza**
I corrected the use of long and short dates for future versions.
I added on "estimated" dates

Now:
now-lv

My propozytion:
propozytion-lv

### CODE REVIEW

File: resources/views/partials/tables/current_table.blade.php [hunk 1]
```diff
@@ -31,23 +31,43 @@
...
+                                    {{ 
+                                        $version->released_at->gt(now())
+                                        ? $version->ends_securityfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')'
+                                        : $version->ends_securityfixes_at->translatedFormat(__('DateLongFormat'))
+                                    }}
                                 @endif
                             </td>
                             <td class="px-6 py-4 text-sm text-gray-500 whitespace-nowrap">
-                                {{ $version->is_lts ? '✓' : '' }}
+                                {{
```
Comment (mattstauffer) [inline resources/views/partials/tables/current_table.blade.php:66]: I'd rather keep this one a single line please

File: resources/views/partials/tables/current_table.blade.php [hunk 2]
```diff
@@ -31,23 +31,43 @@
...
+                                        ? $version->ends_bugfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')'
+                                        : $version->ends_bugfixes_at->translatedFormat(__('DateLongFormat'))
+                                    }}
                                 @endif
                             </td>
                             <td class="px-6 py-4 text-sm text-gray-500 whitespace-nowrap">
                                 @if ($version->ends_securityfixes_at)
-                                    {{ $version->released_at->gt(now()) ? $version->ends_securityfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')' : $version->ends_securityfixes_at->translatedFormat(__('DateLongFormat')) }}
+                                    {{ 
+                                        $version->released_at->gt(now())
```
Comment (mattstauffer) [inline resources/views/partials/tables/current_table.blade.php:59]: Could you indent the second and third line of all of the instances where you changed the syntax like this?

**Review by mattstauffer — CHANGES_REQUESTED** [review]
Good work! A few small notes.

**Comment by olsza** [comment]
I understand wherever I did some lines (good practice) to change back to single line?
If so, no problem ;)
I am asking because I do not know if this applies to all recent changes or only to the two that you marked.

**Comment by olsza** [comment]
ok, corrected :)

**Comment by mattstauffer** [comment]
Thanks for the updates!

**Pull request merged**